### PR TITLE
changed references to

### PR DESCRIPTION
**Pull request by j-c-berger**
Signed-off-by: Jacob Berger <jacob.berger@ibm.com>

For issue 1488: https://github.com/eclipse/codewind/issues/1488

Minor changes to doc content. Given PR 261 for 1178, I changed the reference to: `image-registry.openshift-image-registry.svc:5000` to align with the doc's new name. 

@maysunfaisal and/or @johnmcollier, ready for your review.

### CODE REVIEW

**Comment by j-c-berger** [comment]
@maysunfaisal, thanks for the clarification. I thought since we were holding off on adding new screenshots, we'd keep the existing screenshots, but I understand now the depicted UI is incorrect. See my new commit for the changes. 

For merging the instructions, I added a new heading, 'OpenShift registry in Che and Codewind' and made 'Adding OpenShift in Che` and `Adding OpenShift in Codewind` subheadings. I thought this method would group them together but still divide them enough to avoid confusion. Let me know what you think and thanks for all your help.

File: docs/_documentations/openshiftregistry.md [hunk 1]
```diff
@@ -66,25 +66,23 @@ Some of the following instructions were adapted from [Remotely Push and Pull Con
...
-   ![Screen Shot 2019-08-22 at 12 19 38 PM](https://user-images.githubusercontent.com/6880023/63531755-32c08000-c4d7-11e9-8fae-74799b50d549.png)
 4. Click **Add**.
-   ![Screen Shot 2019-08-22 at 12 19 41 PM](https://user-images.githubusercontent.com/6880023/63531760-348a4380-c4d7-11e9-8fa8-5cef7fbdcac0.png)
 
-### Adding the OpenShift registry in Codewind
+#### Adding the OpenShift registry in Codewind
 After the internal Docker registry credentials are added to Che as a Docker registry, you can use the credentials in Codewind.
 1. Create or open a Codewind workspace.
-   - **Note:** If you're opening an existing workspace, you must stop and start the workspace before the registry credentials take effect.
+   - **Note:** If you open an existing workspace, you must stop and start the workspace before the registry credentials take effect.
```
Comment (maysunfaisal) [inline docs/_documentations/openshiftregistry.md]: This Note is no longer true. You may delete it

File: docs/_documentations/openshiftregistry.md [hunk 2]
```diff
@@ -66,25 +66,23 @@ Some of the following instructions were adapted from [Remotely Push and Pull Con
...
 2. Run the `Codewind: Set Deployment Registry` command.
```
Comment (maysunfaisal) [inline docs/_documentations/openshiftregistry.md]: This is no longer the command. It is called `Codewind: Image Registry Manager`.

File: docs/_documentations/openshiftregistry.md [hunk 3]
```diff
@@ -66,25 +66,23 @@ Some of the following instructions were adapted from [Remotely Push and Pull Con
...
 1. Create or open a Codewind workspace.
-   - **Note:** If you're opening an existing workspace, you must stop and start the workspace before the registry credentials take effect.
+   - **Note:** If you open an existing workspace, you must stop and start the workspace before the registry credentials take effect.
 2. Run the `Codewind: Set Deployment Registry` command. 
-   ![Screen Shot 2019-08-22 at 12 53 19 PM](https://user-images.githubusercontent.com/6880023/63533949-e592dd00-c4db-11e9-99d6-906a8c5edf21.png)
-3. Enter `image-registry.openshift-image-registry.svc:5000/<project>` as the deployment registry, where `<project>` is the OpenShift project that you created the service account in.
-   - Example: `image-registry.openshift-image-registry.svc:5000/pushed`
-   ![Screen Shot 2019-08-22 at 12 53 24 PM](https://user-images.githubusercontent.com/6880023/63534021-1c68f300-c4dc-11e9-937f-a3903e12456d.png)
+3. Enter `docker-registry.default.svc:5000/<project>` as the deployment registry, where `<project>` is the OpenShift project that you created the service account in.
```
Comment (maysunfaisal) [inline docs/_documentations/openshiftregistry.md]: We should then say

1. Enter `docker-registry.default.svc:5000` as the `Address`(`image-registry.openshift-image-registry.svc:5000` for OCP version 4)
2. Enter the service account name as the `username`.
3. Enter the token retrieved from the service account token secret as the `password`.
4. Enter `<project>` as `namespace`, where `<project>` is the OpenShift project that you created the service account in.

So basically section `Adding the OpenShift registry in Che` is merged here..

File: docs/_documentations/openshiftregistry.md [hunk 4]
```diff
@@ -66,25 +66,23 @@ Some of the following instructions were adapted from [Remotely Push and Pull Con
       ```
 6. Copy the value from the **token** field.
 
-### Adding the OpenShift registry in Che
+### OpenShift registry in Che and Codewind
+
+#### Adding the OpenShift registry in Che
```
Comment (maysunfaisal) [inline docs/_documentations/openshiftregistry.md]: Pls delete section `Adding the OpenShift registry in Che`, see bottom review comment where i merged this info

File: docs/_documentations/openshiftregistry.md [hunk 5]
```diff
@@ -66,25 +66,23 @@ Some of the following instructions were adapted from [Remotely Push and Pull Con
       ```
 6. Copy the value from the **token** field.
 
-### Adding the OpenShift registry in Che
+### OpenShift registry in Che and Codewind
```
Comment (maysunfaisal) [inline docs/_documentations/openshiftregistry.md]: I think we can remove `Che` from the title since we are no longer doing it on Che

**Comment by j-c-berger** [comment]
@johnmcollier, please see my new commit, `merged instructions` and if all looks good to you, we can merge and close this PR. Thanks!

File: docs/_documentations/openshiftregistry.md [hunk 6]
```diff
@@ -66,25 +66,12 @@ Some of the following instructions were adapted from [Remotely Push and Pull Con
...
-3. Enter the following information:
-   - Enter `docker-registry.default.svc:5000` as the address.
-   - Enter the service account name as the username.
-   - Enter the token retrieved from the service account token secret as the password.
-   ![Screen Shot 2019-08-22 at 12 19 38 PM](https://user-images.githubusercontent.com/6880023/63531755-32c08000-c4d7-11e9-8fae-74799b50d549.png)
-4. Click **Add**.
-   ![Screen Shot 2019-08-22 at 12 19 41 PM](https://user-images.githubusercontent.com/6880023/63531760-348a4380-c4d7-11e9-8fa8-5cef7fbdcac0.png)
-
 ### Adding the OpenShift registry in Codewind
 After the internal Docker registry credentials are added to Che as a Docker registry, you can use the credentials in Codewind.
```
Comment (johnmcollier) [inline docs/_documentations/openshiftregistry.md]: This line is no longer applicable

**Comment by j-c-berger** [comment]
@johnmcollier, thanks for the review. New commit reflects the change. Let me know if it's ready to merge and close.
